README.md

```diff
@@ -1,4 +1,201 @@
-# allcontributors-to-zenodo
+# allcontributors-zenodo (a2z)
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+## What is a2z?
+
+A2z (All contributors to zenodo) is a tool and GitHub action that will allow you to:
+
+ - generate or update an [all-contributors](https://github.com/all-contributors) entry.
+ - add contributors to the all-contributors file via finding them via the GitHub API
+ - updating a .zenodo.json file with these same individuals
+
+To retrieve repository contributors, we use the GitHub API in both cases.
+**Under development** we will look for Orcid identifiers to go along with 
+associated emails, and add them to the Zenodo.json, along with keeping
+a cache.
+
+## GitHub Action
+
+Since [all-contributors](https://github.com/all-contributors) requires node,
+you might find it easiest to interact with the tool via a GitHub action.
+You can see examples in the [examples](examples) folder.
+
+### Inputs
+
+| name | description | required | default |
+|------|-------------|----------|---------|
+| zenodo_file | .zenodo.json to update. If does not exist, must define zenodo_doi | false | .zenodo.json | 
+| zenodo_doi | Zenodo DOI needed for init. Leave unset to skip init. | false | unset | 
+| log_level | Log level to use, one of INFO, DEBUG, CRITICAL, ERROR, WARNING, FATAL (default INFO) | false | INFO | 
+| threshold | the minimum number of contributions required to add a user | false | 1 | 
+| force | if files exist, force overwrit | false | false |
+| ctype |Contribution type, which defaults to "code" if not set. | false | code |
+| skip_generate | skip running all-contributors generate | false | false |
+
+If you aren't familiar with all-contributors, you'll need to add some
+[commenting in your repository README](https://allcontributors.org/docs/en/cli/usage)
+so the client knows where to write.
+
+## Docker Usage
+
+If you don't want to use the GitHub Action but want to interact with the tool
+in a container, you can do that too! 
```

Comment:
well -- I thought that a regular CLI tool which could be uploaded to pypi or packaged in other distributions, would be the way to use it (at minimum, unless the bot would come about, although github action could indeed be of help too)

Reply:
It could be. In fact this is my preference too. But since we have this super annoying npm / yarn install, for a lot of users jumping into a container will be far easier.